executor/grant_test.go

```diff
@@ -176,26 +176,26 @@ func (s *testSuite) TestColumnScope(c *C) {
 
 func (s *testSuite) TestIssue2456(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
-	tk.MustExec("CREATE USER 'dduser'@'%' IDENTIFIED by '123456';")
-	tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.* TO 'dduser'@'%';")
-	tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.`te%` to 'dduser'@'%';")
+	tk.MustExec("CREATE USER 'dduser'@`%` IDENTIFIED by '123456';")
```

Comment:
why change this?

Reply:
You modified it wrong. You should modify the quotation marks for the SQL statement instead of the quotation marks for %.

Reply:
@winkyao @yangwenmai I don't understand why changing the backquote to single quotes would report the following error. So keep the original backquote.

```
FAIL: grant_test.go:177: testSuite.TestIssue2456

grant_test.go:180:
    tk.MustExec(`GRANT ALL PRIVILEGES ON 'dddb_%'.* TO 'dduser'@'%';`)
/Users/marong/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:160:
    tk.c.Assert(err, check.IsNil, check.Commentf("sql:%s, %v, error stack %v", sql, args, errors.ErrorStack(err)))
... value *errors.Err = &errors.Err{message:"", cause:(*errors.errorString)(0xc4201c2d40), previous:(*errors.Err)(0xc4205d2730), file:"/Users/marong/go/src/github.com/pingcap/tidb/util/testkit/testkit.go", line:134} ("line 1 column 32 near \".* TO 'dduser'@'%';\" (total length 51)")
... sql:GRANT ALL PRIVILEGES ON 'dddb_%'.* TO 'dduser'@'%';, [], error stack line 1 column 32 near ".* TO 'dduser'@'%';" (total length 51)
```

Reply:
Just keep the original code is ok:

```
tk.MustExec("CREATE USER 'dduser'@'%' IDENTIFIED by '123456';")
tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.* TO 'dduser'@'%';")
tk.MustExec("GRANT ALL PRIVILEGES ON `dddb_%`.`te%` to 'dduser'@'%';")
```

Reply:
@winkyao The code have changed to original, and `make dev` successfully on local, but check failed on github.

Reply:
@winkyao `executor/grant_test.go` have been changed to original, thx.